modules/aws/worker-asg/worker-asg.tf

```diff
@@ -24,8 +24,9 @@ resource "aws_cloudformation_stack" "worker_asg" {
         "LoadBalancerNames": [
           "${var.cluster_name}-worker"
         ],
-        "MaxSize": "${var.worker_count}",
-        "MinSize": "1",
+        "MaxSize": "${var.worker_count + 1}",
+        "DesiredCapacity": "${var.worker_count}",
+        "MinSize": "${var.worker_count}",
```

Comment:
Does that mean we'll run one more instance than specified? Isn't it more explicit to just add one node to all AWS control planes?

Reply:
I understand this that `min == number of workers normally`, `desired == min` and `max == desired + 1` which allows to scale by one during updates?

Reply:
>  Isn't it more explicit to just add one node to all AWS control planes?

It's more expensive to always run 4 VMs.

> I understand this that min == number of workers normally, desired == min and max == desired + 1 which allows to scale by one during updates?

Yes.

Reply:
Ah, yes. Me dumb.